app/assets/javascripts/adjustments.js

```diff
@@ -0,0 +1,46 @@
+// Place all the behaviors and hooks related to the matching controller here.
+// All this logic will automatically be available in application.js.
+// You can use CoffeeScript in this file: http://coffeescript.org/
+
+const item_option = item =>
+  `<option value='${item.item_id}'> \
+${item.item_name} -- ${item.quantity} \
+</option>`;
```

Comment:
Some of this  a bit hard to read.

Reply:
We can do some cleanup, no worries. I want to do some restructuring anyways, now that it's homogenized